docs/GOOGLE_OAUTH_SUBMISSION_GUIDE.md

```
@@ -0,0 +1,350 @@
+
+**PT (referencia):**
+
+> AICA (Life OS) e uma plataforma de gestao de vida pessoal para brasileiros. Integra com Google Calendar para sincronizacao bidirecional de eventos com o sistema de gerenciamento de tarefas — usuarios gerenciam sua vida na AICA e veem refletido no Google Calendar automaticamente. A integracao usa o escopo minimo necessario (calendar.events) e usuarios podem desconectar a qualquer momento.
+
+---
+
+## 4. Como os Escopos Serao Usados (How Will Scopes Be Used)
+
+Esta secao responde diretamente a pergunta do formulario Google.
+
+### calendar.events
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)